src/views/Alteration.vue

```diff
@@ -8,8 +8,8 @@
         </header>
 
         <section class="mt-6">
-          You are legally obligated to keep your company information up to date. Necessary fees
-          will be applied as updates are made.
+          You are legally obligated to keep your company information up to date. Necessary fees will be
+          applied as updates are made.
```

Comment:
This (and others below) is just to make the max width somewhere between 100 and 120 chars.